core/lib/Pdf/Core/Parsers/Object.hs

```diff
@@ -24,9 +24,9 @@ where
 import Pdf.Core.Object
 import qualified Pdf.Core.Name as Name
 import Pdf.Core.Parsers.Util
+import Prelude
```

Comment:
Why are you importing it explicitly? I don't think we have `NoImplicitPrelude` enabled...

Reply:
> Why are you importing it explicitly? I don't think we have `NoImplicitPrelude` enabled...

Oops!. Fixed.